cli/pkg/kctrl/local/reconciler.go

```diff
@@ -44,10 +44,11 @@ func NewReconciler(depsFactory cmdcore.DepsFactory,
 }
 
 type ReconcileOpts struct {
-	Local     bool
-	KbldBuild bool
-	Delete    bool
-	Debug     bool
+	Local           bool
+	KbldBuild       bool
+	Delete          bool
+	Debug           bool
+	DeployResources bool
```

Comment:
There are get/list commands as well where we need connection but we don't deploy resources. Maybe we can simply use `coreClient` or `clientSet` or something similar?

Reply:
> There are get/list commands as well where we need connection but we don't deploy resources.

Elaborate on what you are thinking about?
From my perspective, `Deploy` is true only when `kapp` (or the deploy section) is going to be used.

Are you trying to say that we also use it to grab service accounts? That is how we "configure" kapp from my perspective

Reply:
Never mind, I mixed up things. Just re read the issue and I think this is fine.